casanovo/casanovo.py

```diff
@@ -126,14 +126,16 @@ def main(
         max_mz=float,
         min_intensity=float,
         remove_precursor_tol=float,
+        max_charge=int,
+        precursor_mass_tol=float,
+        isotope_error_range=lambda min_max: (int(min_max[0]), int(min_max[1])),
```

Comment:
**question:** Is there a reason to use a range here instead of just specifying the maximum isotope error?

Reply:
The range makes it possible to not only look at a previous isotopic peak (subtract C13 mass difference), but also at a next isotopic peak (add C13 mass difference). Most of the time this is probably not necessary, but for some molecules it can be that the mono-isotopic peak is in the middle of the isotopologue series. See e.g. [here](https://pubs.acs.org/doi/10.1021/jasms.1c00110).

It's an edge case though, so maybe this is overkill. Should we simplify it to only look backwards?

Reply:
I think it's fine to leave it how it is. Thanks!